### PR TITLE
Remove host abstraction

### LINKED ISSUE #87: Is the host abstraction necessary?
There's a bit of abstraction about the concept of "portal host", which with the current spec doesn't seem to buy that much. Saying that there's a generic portal host concept, which HTMLPortalElement implements, seems to abstract only over "accept a message posted to the host".

Are there plans for future types of hosts that would make this abstraction more worthwhile? Is it just that much cleaner? It's unclear to me, at least.

### PR DESCRIPTION
**Pull request by jeremyroman**
Resolves #87 by simply assuming that the only type of host is `HTMLPortalElement`. In particular this allows `PortalHost/postMessage` to be one contiguous block.

We can unwind this if we ever do care about some other kind of host environment.

***
<a href="https://pr-preview.s3.amazonaws.com/jeremyroman/portals/pull/127.html" title="Last updated on May 17, 2019, 8:19 PM UTC (e77e9ea)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/portals/127/665046f...jeremyroman:e77e9ea.html" title="Last updated on May 17, 2019, 8:19 PM UTC (e77e9ea)">Diff</a>

### CODE REVIEW

**Comment by jeremyroman** [comment]
@domenic wdyt?

**Comment by jeremyroman** [comment]
This might also address #69. @domenic?

**Comment by jeremyroman** [comment]
@lucasgadani ptal

File: index.bs [hunk 1]
```diff
@@ -215,8 +220,9 @@ spec:url; type:dfn; text:scheme
     portals-rendering.html
   </wpt>
 
-  A <{portal}> element may have a <dfn for="HTMLPortalElement">guest browsing context</dfn>, which is a
-  [=portal browsing context=]. If so, the element is the [=host=] of its [=guest browsing context=].
+  A <{portal}> element has a <dfn for="HTMLPortalElement">guest browsing context</dfn>, which is
```
Comment (lucasgadani) [inline index.bs]: This is hard to read. Can we simplify it somehow?

Comment (jeremyroman) [reply index.bs]: Introduced a variable to replace the pronoun "it". English is so terrible for when I really want to say "GuestBrowsingContext = HostElement<sup>-1</sup>".

**Pull request merged**